angular.json

```diff
@@ -629,8 +635,11 @@
           }
         },
         "test": {
-          "builder": "@angular-devkit/build-angular:karma",
+          "builder": "@angular-builders/custom-webpack:karma",
           "options": {
+            "customWebpackConfig": {
+              "path": "./extra-webpack.setup.config.js"
+            },
```

Comment:
Instead of creating a new global `extra-webpack.config.js`, that is used for all test runs,
can we simply add a `fallback` section with  `url: false` only in [`karma.conf.js` of the setup lib](https://github.com/SAP/spartacus/blob/develop/core-libs/setup/karma.conf.js#L46-L49)?

Reply:
Update: As discussed offline, the `karma.conf.js` doesn't seem to respect the additional property `node: {global: true}`.

Reply:
Yep, and it will be remove in https://github.com/SAP/spartacus/pull/16532/files